opencodelists/urls.py

```diff
@@ -4,6 +4,7 @@
 from django.views.generic import RedirectView
 
 from . import views
+from .views.errors import bad_request, page_not_found, permission_denied, server_error
 
 
```

Comment:
To cause the error pages to be used when the errors are triggered:

```suggestion
from django.conf.urls import handler404, handler500, handler403, handler400
handler400 = bad_request
handler403 = permission_denied
handler404 = page_not_found
handler500 = server_error
```
Note that Django debug toolbar will still take precedence if installed so set DEBUG to False if you want to provoke them to see them work locally.

Reply:
```python
from .views.errors import csrf_failure
CSRF_FAILURE_VIEW = csrf_failure
```
In opencodelists/settings.py will cause the CSRF handler to take effect. Then I think this is good to go.

I think this fixes #2168 and should be linked as such.

Reply:
Templates and views all look good.